base/server/python/pki/server/deployment/__init__.py

```diff
@@ -760,37 +760,20 @@ def setup_system_certs(self, subsystem, client):
 
     def load_admin_cert(self, subsystem):
 
-        logger.info(
-            'Loading admin cert from client database: %s',
-            self.mdict['pki_admin_nickname'])
-
-        client_nssdb = pki.nssdb.NSSDatabase(
-            directory=self.mdict['pki_client_database_dir'],
-            password=self.mdict['pki_client_database_password'])
-
-        try:
-            b64cert = client_nssdb.get_cert(
-                nickname=self.mdict['pki_admin_nickname'],
-                output_format='base64',
-                output_text=True,  # JSON encoder needs text
-            )
-
-        finally:
-            client_nssdb.close()
-
-        if b64cert:
-            return b64cert
+        logger.info ('load_admin_cert')
 
         standalone = config.str2bool(self.mdict['pki_standalone'])
         external = config.str2bool(self.mdict['pki_external'])
+        external_step_two = config.str2bool(self.mdict['pki_external_step_two'])
 
-        if standalone or external and subsystem.type in ['KRA', 'OCSP']:
+        if external_step_two and subsystem.type!='CA':
```

Comment:
The `standalone` and `external` variables are no longer used. `pylint` requires that those variables be removed.